static/app/views/settings/project/projectKeys/details/loaderSettings.tsx

```diff
@@ -225,8 +225,15 @@ export function LoaderSettings({keyId, orgSlug, project, data, updateData}: Prop
               !sdkVersionSupportsPerformanceAndReplay(data.browserSdkVersion)
                 ? t('Only available in SDK version 7.x and above')
                 : data.dynamicSdkLoaderOptions.hasReplay
-                  ? t(
-                      'When using Replay, the loader will load the ES6 bundle instead of the ES5 bundle.'
+                  ? tct(
+                      'When using Replay, the loader will load the ES6 bundle instead of the ES5 bundle. The default configurations are [codeReplay:replaysSessionSampleRate: 0.1] and [codeError:replaysOnErrorSampleRate: 1]. To learn how to configure this, click [configDocs:here].',
```

Comment:
more in line with what we typically phrase the docs links
```suggestion
                  ? tct(
                      'When using Replay, the loader will load the ES6 bundle instead of the ES5 bundle. The default configurations are [codeReplay:replaysSessionSampleRate: 0.1] and [codeError:replaysOnErrorSampleRate: 1]. [configDocs:Read the docs] to learn how to configure this.',
```

Reply:
thank you!!